setup.py

```diff
@@ -2,7 +2,7 @@
 
 from setuptools import setup, find_packages
 
-__version__ = '1.0.19'
+__version__ = '1.1.0'
```

Comment:
Any reason that this kind of change becomes a minor release as opposed to a patch?

Reply:
No particularly for these PRs it’s even superdebatable - because the changes only affect Docker users, but nothing new for PyPi package users. 

But I think Amundsen is generally erring on the “to many patches, to few minor changes”. Not sure if a policy was ever discussed (i.e. are we doing semantic versioning?) My understanding is patches are “no new features - just correction of bugs”. So if it does some new thing you bump minor. Happy to be enlightened though, not a super strong point for me!

So, for this particular change my thinking was: *”it adds a new feature to rely on for the compose file, so it’s a minor feature not just a fix/patch”*

Reply:
I c, currently we don't doing semantic versioning as Lyft internally relies on import the library version to pick up the change. We could have a discussion later if semantic versioning is sth we aim for.

Also feel free to ignore my comment on the search pr. I think it make sense to bump the minor version per your comment.

Reply:
👍 that was a quick discussion! Agree it’s just numbers (and there are plenty to use + not that important if you always pin where you use stuff)

Reply:
> So, for this particular change my thinking was: _”it adds a new feature to rely on for the compose file, so it’s a minor feature not just a fix/patch”_

I can see that. Sounds good. Yes we're using semantic versioning.

Reply:
Hm, okay guess we're not using semantic versioning here. 😅 It's what we're aiming for on amundsenfrontendlibrary though, because if we don't make the distinction of bug vs new feature vs backwards incompatible change on the frontend -- people will end up getting really confused and unable to get the app to work. Community meeting/slack channel topic?

Reply:
😆 there! Now I started getting you Lyft’ers all confused about what you’re *actually doing* (honestly not my intention, I swear) If y’all up for it we can certainly open it up OTOH we should  watch out for the “semantic police” (though I’ve only seen sensible people around here so far) ... I’ll let you Lyft folks decide and take the lead on it.